ecommerce-ui/src/App.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+import RentalList from './RentalList';
+import data from './airbnbs.json';
+
+import { configure } from 'enzyme';
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
+});
+
+const junk = ()=>{
+  console.log('test');
+}
+
+describe("RentalList", ()=> {
+  const wrapper = shallow(<RentalList rentals={data}
```

Comment:
To access this `shallow` component, you'll want to import it from `enzyme`. Then create an `it` block within the `describe` and write the test inside there.